Example/HelloWorld/Package.swift

```diff
@@ -0,0 +1,43 @@
+// swift-tools-version: 5.9
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "HelloWorld",
+    platforms: [
+        .macOS(.v13),
+        .iOS(.v16),
+    ],
+    products: [
+        .library(
+            name: "HelloWorldLib",
+            targets: ["HelloWorldLib"]
+        ),
+        .library(
+            name: "TodoModels",
+            targets: ["TodoModels"]
+        ),
+    ],
```

Comment:
Is this target re-declaration necessary here? It would be good if all we needed to do was provide the dependencies field below.